Add meeting agenda 2024-04-03 (#644)
solid/specification: meetings/2024-04-03.md

# W3C Solid Community Group: Weekly

* Date: 2024-04-03T14:00:00Z
* Call: https://meet.jit.si/solid-cg
* Chat: https://matrix.to/#/#solid_specification:gitter.im
* Repository: https://github.com/solid/specification

> * Status: Draft

Reviewer: ```suggestion
* Status: Published
```

* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! please introduce yourself.

### Scribes
* Sarven Capadisli

### Introductions
* KS: Interested about the work you're doing. Going to pay attention to some of the decisions you come up with.

---

## Topics

### Implementation Feedback

* RG: I've put out a middleware for Solid-PREP. Feel free to check it out.
    * https://github.com/CxRes/express-accept-events-temp
    * https://github.com/CxRes/express-prep-temp
    * https://github.com/CxRes/express-events-negotiate-temp

* RG: These are connect/express middlewares that work together to implement PREP. 

### discussion#555 should be made to look like it's no longer active, with more blatant link to current "project"16
URL: https://github.com/solid/specification/issues/635

* VB: Related: https://github.com/w3c-cg/solid/issues/20

> * RG: We need to migrate the discussion over?

Reviewer 2: ```suggestion
* TT: Related: https://github.com/orgs/solid/projects/16
* RG: We need to migrate the discussion over?
```

* MdJ: The way to propose a STM is to put it on the project board.
* RG: It was about the minutes. How/where.

> * eP: I think we can worry about it when we have minutes. When only 3 people joining and questionable if anyone is scribing, one of the people may write a summary. STM hasn't been working so well — low participation.

Reviewer 2: ```suggestion
* eP: I think we can worry about it when we have minutes. When only 3 people join, and its questionable if anyone is scribing, one of the people might write a summary. STM hasn't been working so well — low participation.
```

* RG: We've been keeping minutes sometimes. SC was pointing out having it accessible from one place. Not hackmd. Sarven's proposal was to PR but I felt it was a bit too much. In the discussion, there is a way to record. VB pointed this out last week. Whatever we do, even if in hackmd, we can copy over so that everything is in one place.

> * MdJ: There is a column in project 16. That's the right place to list it. Use URLs that would be persisted. For instance GH.

Reviewer 2: ```suggestion
* MdJ: There is a column in [project 16](https://github.com/orgs/solid/projects/16). That's the right place to list it. Use URLs that would be persisted. For instance GH.
```

* RG: When you open that, there is a place where you can record text.
* MdJ: Should have hyperplinks.
* HZ: If you want things in one place, doesn't matter.
* SC: Separate the idea on scratchpad notes from minutes. The latter is imporant for referencing, decisions, etc., and they go in [solid/specification](https://github.com/solid/specification/).

### Meeting Minutes

* VB: We've been operating with the understanding that the minutes are available relatively soon after the meeting (for several years in fact). [CONTRIBUTING](https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md#decisions) indicates that expectation to merge "within 2 days." Do we need to change this expectation to 1 month? 
* VB: If there is an issue with the pipeline we are currently using, the alternative to use W3C infrastructure could allow us to publish minutes immediately after the call: <https://github.com/w3c-cg/solid/issues/18>.
* MdJ: Does anyone know what VB wants to say about this?
* eP: I think the understanding is the chair should make a PR soon after the meeting and merge after two days. Concern is if PR is not made and/or it is not merged.

> * MdJ: Rather than guessing what VB is thinking, lets move on to the next topic.

Reviewer 2: ```suggestion
* MdJ: Rather than guessing what VB is thinking, let's move on to the next topic.
```
